src/transformState.js

```diff
@@ -5,7 +5,29 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+
+  Object.assign(state, state);
+
+  for (const action in actions) {
+    switch (actions[action].type) {
```

Comment:
would be nice to create a variable for `actions[action]` - code will be more readable in this case

Reply:
done